docs/api/apiv3/paths/workspaces.yml

```
@@ -0,0 +1,95 @@
+# /api/v3/workspaces
+---
+get:
+  summary: List workspace
+  operationId: list_workspace
+  tags:
+    - Workspace
+  description: Returns a collection of workspace. The collection can be filtered via
+    query parameters similar to how work packages are filtered. In addition to the
+    provided filter, the result set is always limited to only contain workspace the
+    client is allowed to see.
+
+    Since workflows are the generic term for a number of resources like projects and portfolios, the returned
+    collection will contain a mix of those resources.
+  parameters:
+    - name: filters
+      schema:
+        type: string
+      in: query
+      required: false
+      description: |-
+        JSON specifying filter conditions.
+        Accepts the same format as returned by the [queries](https://www.openportfolio.org/docs/api/endpoints/queries/) endpoint.
+        Currently supported filters are:
+
+        + active: based on the active property of the portfolio
+        + ancestor: filters workspace by their ancestor. A portfolio is not considered to be its own ancestor.
+        + available_project_attributes: filters workspace based on the activated project attributes.
+        + created_at: based on the time the portfolio was created
+        + favorited: based on the favorited property of the portfolio
+        + id: based on workspace' id.
+        + latest_activity_at: based on the time the last activity was registered on a portfolio.
+        + name_and_identifier: based on both the name and the identifier.
+        + parent_id: filters workspace by their parent.
+        + principal: based on members of the portfolio.
+        + project_phase_any: based on the project phases active in a portfolio.
+        + project_status_code: based on status code of the portfolio
```

portfolio -> workspaces


```suggestion
        + active: based on the active property of the workspace
        + ancestor: filters workspaces by their ancestor. A workspace is not considered to be its own ancestor.
        + available_project_attributes: filters workspaces based on the activated project attributes.
        + created_at: based on the time the portfolio was created
        + favorited: based on the favorited property of the workspace
        + id: based on workspace id.
        + latest_activity_at: based on the time the last activity was registered on a workspace.
        + name_and_identifier: based on both the name and the identifier.
        + parent_id: filters workspace by their parent.
        + principal: based on members of the workspace.
        + project_phase_any: based on the project phases active in a workspace.
        + project_status_code: based on status code of the workspace
```